srcpkgs/docker-compose/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'docker-compose'
 pkgname=docker-compose
-version=1.26.2
+version=1.27.0
 revision=1
 archs=noarch
```

Comment:
`noarch` is deprecated, please remove this line.

Reply:
Done.